pkg/collector/corechecks/cluster/orchestrator/collectors/collector.go

```diff
@@ -90,10 +92,11 @@ func (cm CollectorMetadata) FullName() string {
 // CollectorRunConfig is the configuration used to initialize or run the
 // collector.
 type CollectorRunConfig struct {
-	APIClient   *apiserver.APIClient
-	ClusterID   string
-	Config      *config.OrchestratorConfig
-	MsgGroupRef *atomic.Int32
+	APIClient             *apiserver.APIClient
+	ClusterID             string
+	Config                *config.OrchestratorConfig
+	MsgGroupRef           *atomic.Int32
+	ManifestCollectionCfg *ManifestBufferConfig
```

Comment:
this name implies that the config relates to the overall manifest collection, but the struct implies that it is scoped to the buffer part. Can we make that distinction, if there is one, clear?

Reply:
Thanks for the review I renamed `ManifestCollectionCfg` to `ManifestBufferCfg `